tests/Admin/Admin_Page_Tests.php

```diff
@@ -41,7 +41,9 @@ public function test_add_page() {
 		wp_set_current_user( $admin_user );
 		set_current_screen( 'dashboard' );
 
-		$page_hook    = $this->admin_page->add_page();
+		$this->admin_page->add_and_initialize_page();
+		$page_hook = $this->admin_page->get_hook_suffix();
+		$this->assertNotNull( $page_hook );
```

Comment:
Based on that this value should always be a string:
```suggestion
		$this->assertNotEmpty( $page_hook );
```